.github/workflows/android-release.yml

```diff
@@ -22,22 +26,21 @@ jobs:
         uses: gradle/gradle-build-action@v2
       - name: Build with Gradle
         run: ./gradlew build
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/setup-node@v4
         with:
-          tag_name: ${{ github.ref }}
-          release_name: Debug Release ${{ github.ref }}
-          prerelease: true
-      - name: Upload Debug APK
-        id: upload_release_asset
-        uses: actions/upload-release-asset@v1.0.1
+          node-version: 20
+      - name: Install
+        run: npm install --save @semantic-release/changelog @semantic-release/git @semantic-release/github
+      - name: Semantic Release
+        run: npx semantic-release --dry-run
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Merge Release -> Main
+        uses: devmasx/merge-branch@854d3ac71ed1e9deb668e0074781b81fdd6e771f
```

Comment:
Would it be worth it to just fork the merge-branch action, create a release for it and point to that one instead of pointing to a commit hash? Presumably you did this because the repo is old, the features you want are probably merged to master but not in a release?

Reply:
Their documentation points to `master` so this is a bit safer option. We could take it over or find an alternative

Reply:
Okay, it's up to you